experimental/add_to_app/ios_fullscreen/IOSFullScreen/ViewController.swift

```diff
@@ -6,10 +6,43 @@ import UIKit
 import Flutter
 
 class ViewController: UIViewController {
```

Comment:
Does `GeneratedPluginRegistrant.register(with: self.flutterEngine);` need to be added somewhere now that there's a provider plugin?

Reply:
Provider is just a package, rather than a plugin with native code of its own, so I don't believe this is necessary (correct me if I'm wrong, though!).

I *do* have a separate flutter module in the repo called `flutter_model_using_plugin` that will be used in a different iOS app, and that one will need to do whatever setup is necessary to use plugins.